3_javascript/javascript1/week2/script2.js

```diff
@@ -0,0 +1,181 @@
+// Flight booking fullname function Part !!
+
+function getFullname(firstname, surname) {
+  let fullname = firstname + surname;
+  return fullname;
+}
+
+let fullname1 = getFullname("Milad ", "Fetrat");
+let fullname2 = getFullname("Amin ", "Hussaini");
+
+console.log("Full Name 1:", fullname1);
+console.log("Full Name 2:", fullname2);
+
+
+
+  // Formal fullname Part //
+  function getFullname(firstname, surname, useFormalName, isFemale) {
+    if (useFormalName) { return "Lord " + firstname + surname; }
+    else if (!firstname && !surname) { return "Please provide a valid name."; }
+    else if (isFemale) { return "Lady " + firstname + surname; }
+    else { return firstname + surname; }
+  }
+  
+  const formalFullName = getFullname("Milad ", "Fetrat", true, false);
+  const inFormalFullName = getFullname("Amin ", "Hussaini", false, false);
+  const formalFullFemaleName = getFullname("Hanna ", "Jensen", false, true);
+  const missingNames = getFullname("", "", false);
+  
```

Comment:
There is an issue with this function. If I call the function with setFormalName to true, like you do here
```
const formalFullName = getFullname("Milad ", "Fetrat", true, false);
````
It will not know if it is a female or a male because the function is going to return right away and will not continue with the following conditions.